config/routes.rb

```diff
@@ -4,9 +4,14 @@
   # Defines the root path route ("/")
   # root "articles#index"
   root "users#index"
+
+  get '/posts/new', to: 'posts#new', as: 'create_post'
+  post '/posts/new', to: 'posts#create'
+  get '/users/:user_id/posts/:id', to: 'posts#show', as: 'post'
+  post '/users/:user_id/posts/:id/', to: 'posts#create_comment'
+  post '/users/:user_id/posts/:id/likes', to: 'posts#create_like', as: 'like'
+
```

Comment:
- [ ] Similarly, kindly ensure that there is a route in the `config/routes.rb` file that corresponds with the `create` actions in the likes_controller. This route in the command line should be similar to:
```ruby
  `POST /posts/:post_id/likes(.:format) likes#create`

Reply:
Thank  you sir :pray: